src/js/edu-benefits/reducers/uiState/index.js

```diff
@@ -0,0 +1,182 @@
+import _ from 'lodash/fp';
+
+import {
+  UPDATE_COMPLETED_STATUS,
+  UPDATE_INCOMPLETE_STATUS,
+  UPDATE_REVIEW_STATUS,
+  UPDATE_VERIFIED_STATUS,
+  UPDATE_SUBMISSION_STATUS,
+  UPDATE_SUBMISSION_ID,
+  UPDATE_SUBMISSION_TIMESTAMP
+} from '../../actions';
+
+const ui = {
+  submission: {
+    status: false,
+    errorMessage: false,
+    id: false,
+    timestamp: false
+  },
+  sections: {
```

Comment:
If other reviewers are wondering where these came from (like I was), they map to the mockups: https://classic.moqups.com/greg@adhocteam.us/l32hJ36b/p:a259395ea